### DISCUSSION TITLE
Dark Mode and Router Links

### DISCUSSION THREAD

**Discussion by lisadejonghe** [quasarframework/quasar#12743] [General - Components / Directives / etc] [answered]
When using a router link, for example ` <router-link to="/signup">Sign Up</router-link>` , you automatically get a layout for a link, which means your text gets underlined and becomes blueish/purple. However if you do not want this, you can add `style="text-decoration: none; color: inherit"`.  This is not compatible with dark mode switching, if you are in dark mode, your link color stays white, even if you switch to light mode and the link becomes unreadable.

How can I fix this?

**Comment by MilosPaunovic** [comment, marked as answer]
You can make use of `Dark` plugin: [https://quasar.dev/quasar-plugins/dark](https://quasar.dev/quasar-plugins/dark), meaning add class depending on the activity of the plugin, like this:

```js
<router-link :to="{ name: 'Signup' }" :class="$q.dark.isActive ? 'text-red' : 'text-blue'">Signup</router-link>
```
